src/components/ui/spinner-container/spinner-container.tsx

```diff
@@ -5,7 +5,7 @@ type Props = {
   children: React.ReactNode;
   isLoading: boolean;
 };
-const SpinnerContainer: React.FC<Props> = ({ children, isLoading }) => (
+export const SpinnerContainer: React.FC<Props> = ({ children, isLoading }) => (
```

Comment:
Can you elaborate on why you changed default export for named exports? I'm just curious. I wouldn't argue any of those, but if everything else is coded following that rule, why should we change them all?

Reply:
When we want to navigate through files we have two instances (declaration and exporting). There are parts in the code where we have an export named "ItemDetails" and the import is named "ProductAreaDetails". It's for code consistency.